Samples/ProjectTracker/ProjectTracker.Ui.Blazor.Server/Pages/_Host.cshtml

```diff
@@ -0,0 +1,23 @@
+﻿@page "/"
+@namespace ProjectTracker.Ui.Blazor.Pages
+@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
+@attribute [Microsoft.AspNetCore.Authorization.AllowAnonymous]
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>ProjectTracker.Ui.Blazor</title>
+    <base href="~/" />
+    <link rel="stylesheet" href="css/bootstrap/bootstrap.min.css" />
+    <link href="css/site.css" rel="stylesheet" />
+</head>
+<body>
+  <app>
+    @(await Html.RenderComponentAsync<App>(RenderMode.ServerPrerendered))
```

Comment:
In .NET Core 3.1, I think this can be replaced with use of the <component> tag helper, but I think both work.